content/src/fieldConfig/business-formation.json

```diff
@@ -242,6 +262,23 @@
         "radioYesText": "Yes",
         "error": "Select 'Yes' or 'No'."
       },
+      "isVeteranNonprofit": {
+        "label": "Non-Profit Veteran Corporation",
+        "description": "Is your Domestic Non-Profit a `Veteran Corporation|veteran-corporation`?",
+        "radioNoText": "No",
+        "radioYesText": "Yes",
+        "error": "Select 'Yes' or 'No'.",
+        "domNpVetCorpReviewText": "**Yes**, my Domestic Non-Profit is a Veteran Corporation.",
+        "yesBoardMembersReviewText": "**Yes**, my Non-Profit will have board members.",
+        "noBoardMembersReviewText": "**No**, my Non-Profit will not have board members.",
+        "undefinedBoardMembersReviewText": "**My Non-Profit will or will not have board members**",
```

Comment:
[pebble] Why does this exist? I can see in your code elsewhere that the boardMembers is a required field and undefined is not a valid answer - they won't be allowed to submit if it is undefined. I think this should have the "Not Entered" highlighted text when it is undefined, right?